extensions/new-keccak256/circuit/src/keccakf/columns.rs

```diff
@@ -0,0 +1,38 @@
+use openvm_circuit::system::memory::offline_checker::{MemoryReadAuxCols, MemoryWriteAuxCols};
+use openvm_circuit_primitives_derive::AlignedBorrow;
+use p3_keccak_air::KeccakCols as KeccakPermCols;
+
+#[repr(C)]
+#[derive(Debug, AlignedBorrow)]
+pub struct KeccakfVmCols<T> {
+    pub inner: KeccakPermCols<T>,
+    pub preimage_state_hi: [T; 100],
+    pub postimage_state_hi: [T; 100],
+    pub instruction: KeccakfInstructionCols<T>,
+    pub mem_oc: KeccakfMemoryCols<T>,
+    pub timestamp: T,
+    pub is_enabled_is_first_round: T,
+    pub is_enabled_is_final_round: T,
+}
+
+#[repr(C)]
+#[derive(Copy, Clone, Debug, Default, AlignedBorrow, derive_new::new)]
+pub struct KeccakfInstructionCols<T> {
+    pub pc: T,
+    pub is_enabled: T,
+    pub buffer_ptr: T,
+    pub buffer: T,
+    pub buffer_limbs: [T; 4],
```

Comment:
seems a bit ambiguous. how about
```suggestion
    pub buffer_reg_ptr: T,
    pub buffer_ptr: T,
    pub buffer_ptr_limbs: [T; 4],
```